.github/workflows/notebooks.yml

```diff
@@ -0,0 +1,64 @@
+name: notebooks
+
+on:
+  push:
+    paths-ignore:
+      # specific folder locations
+      - ".vscode/**"
+      - "docs/**"
+      # filetypes
+      - "**.md"
+      - "**.rst"
+      - "**.ipynb"
```

Comment:
I don't think we want to ignore these in this case
```suggestion
```

Reply:
Definitely!! Good catch :)